src/app/shared/components/organisms/participant-list/participant-list.component.scss

```
@@ -1,4 +1,5 @@
-@import "main";
+@use "utilities/_colors";
+@use "../../../../../styles/utilities/font-weights";
```

Should `font-weights` be imported with the alias uri as well? Also, remove `_` from partial files' names.

```suggestion
@use "utilities/colors";
@use "utilities/font-weights";
```